test/mml_builder.js

```diff
@@ -656,7 +656,8 @@ suite('mml_builder use_workers=' + useWorkers, function() {
   });
 
   // See https://github.com/Vizzuality/grainstore/issues/62
-  test('throws useful error message on invalid text-name', function(done) {
+  // skiping since carto#0.16.3 does not throw error
+  test.skip('throws useful error message on invalid text-name', function(done) {
```

Comment:
I've just found this: 
https://github.com/mapbox/carto/commit/52343ff833c05185f5118eb0b66cbe5d81ae8dd8
<--
https://github.com/mapbox/carto/issues/297
<--
https://github.com/mapbox/carto/commit/5938ebb6097809ecab103936cfd0100fb74f9755

looks like `invalid` is actually a valid value because is considered an expression. So, I'd say that we shouldn't revert this change. Should we?